sdk/formrecognizer/Azure.AI.FormRecognizer/api/Azure.AI.FormRecognizer.netstandard2.0.cs

```diff
@@ -53,15 +53,79 @@ public enum ServiceVersion
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
         public FormRecognizerLanguage(string value) { throw null; }
+        public static Azure.AI.FormRecognizer.FormRecognizerLanguage Af { get { throw null; } }
```

Comment:
with enums so long... it kind of looses the purpose of being helpful for discoverability.
Could you open an issue on Github for us to triple check that this is ok? and if not, then go back to a string for language?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/19830